app/assets/stylesheets/_globals.sass

```diff
@@ -37,6 +37,9 @@ header
 #root
   @include outer-container
   .app
+    &.invitation-form
+      input
+        display: block
```

Comment:
@hestervanwijk had dit even toegevoegd om tijdelijk het formulier wat mooier te maken. Als het niet meer nodig is mag deze regel wat mij betreft weer weg.